Comanche055/RESTARTS_ROUTINE.agc

```diff
@@ -162,13 +162,13 @@ ITSLIKEB	CA	RTRNCADR	# TYPE B, SO STORE RETURN IN
 
 ITSAJOB		NDX	TEMP2G		# NOW ADD THE PRIORITY AND LET'S GO
```

Comment:
✨ Change `LET'S` to `LET:S`
✨ Also lines `0091` and `0108` that need `'` changed to `:` (see [p1416](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1416.jpg) scans)